### DISCUSSION TITLE
Not detecting Huion HS611

### DISCUSSION THREAD

**Discussion by stonkol** [OpenTabletDriver/OpenTabletDriver#3148] [Q&A] [answered]
Hey, I am on macOS Sonoma and OpenTabletDriver v0.6.4.0 couldn't detect this tablet

**Comment by jamesbt365** [comment, marked as answer]
Please ensure OTD has access to the [required permissions](https://opentabletdriver.net/Wiki/Documentation/RequiredPermissions#macos).

**Reply by stonkol** [reply]
Thanks for replying, it was all ticked, unfortunately still didn't work.
The console is showing:

`[Detect:Info]	No tablets were detected.`
`[Settings:Info] Presets have been refreshed. Loaded 0 presets.`

I tried a clean reinstallation of the driver app and restart the computer, neither works.

**Reply by jamesbt365** [reply]
Export diagnostics (help > export diagnostics) and upload that file here.

**Reply by stonkol** [reply]
Ok, here is the file

[diagnostics_macos_huionhs611.json](https://github.com/OpenTabletDriver/OpenTabletDriver/files/13774000/diagnostics_macos_huionhs611.json)

**Reply by jamesbt365** [reply]
Tablet > Device String Reader
9580
109

Dump all, save that file and send it here.

**Reply by stonkol** [reply]
Ok, today it start working, Cheers
